src/app/home/home.component.html

```diff
@@ -495,16 +495,22 @@
                 }"
             >
                 <div>
-                    <span class="waiting-text"
-                          [ngClass]="{
-                    'waiting-text-dark-mode': darkModeService.isDarkMode
-                }">duke lexuar shkrimin...</span>
+                    <span
+                        class="waiting-text"
+                        [ngClass]="{
+                            'waiting-text-dark-mode': darkModeService.isDarkMode
+                        }"
+                        >duke lexuar shkrimin...</span
+                    >
                 </div>
             </div>
-            <ul class="list-group list-group-flush while-reading-text-ul"
+            <ul
+                class="list-group list-group-flush while-reading-text-ul"
                 [ngClass]="{
-                    'while-reading-text-ul-dark-mode': darkModeService.isDarkMode
-                }">
+                    'while-reading-text-ul-dark-mode':
+                        darkModeService.isDarkMode
+                }"
+            >
```

Comment:
Do these changes serve any purpose? If it is just for indentation, I'd say undo these changes and only push full-file formatting changes e.g. with prettier in a separate PR if you'd like.

Reply:
it's a formatting issue, that was resolved by itself after running the prettier:fix method.

Reply:
ok i will